### PR TITLE
Change the import path to github.com/grafana/pyroscope-go

### PR DESCRIPTION
**Pull request by cyriltovena**
This changes the import path, we'll need to tag it so that users can update manually.

Once tagged it also means the new command to install will be

```
go get github.com/grafana/pyroscope-go
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,43 +2,47 @@
...
 
 ```go
 package main
 
-import "github.com/pyroscope-io/client/pyroscope"
+import "github.com/grafana/pyroscope-golang/profiler"
 
 func main() {
-  pyroscope.Start(pyroscope.Config{
+  profiler.Start(profiler.Config{
```
Comment (kolesnikovae) [inline README.md:23]: I'd like to propose a slightly different approach – to move everything user need to import to the root and change the root package name to "pyroscope": https://github.com/grafana/pyroscope-golang/compare/main...chore/reorganize-package. In this way users won't need to change their code (see [`example`](https://github.com/grafana/pyroscope-golang/compare/main...chore/reorganize-package#:~:text=3-,example/main.go,-%40%40%20%2D3%2C11%20%2B3%2C12%20%40%40%20package)).

Also, in the branch the module is called `pyroscope-go`, which in my opinion is more correct (as the official name is Go) and commonly used:
 - https://github.com/kubernetes/client-go
 - https://github.com/elastic/go-elasticsearch
 - https://github.com/opensearch-project/opensearch-go
 - https://github.com/confluentinc/confluent-kafka-go
 - https://github.com/segmentio/parquet-go

Although, that would require a change of the repo name

Comment (cyriltovena) [reply README.md:23]: I think we could make the repo name change ?

Comment (cyriltovena) [reply README.md:23]: let me prepare everything and see.

**Review by kolesnikovae — APPROVED** [review]
LGTM!

**Pull request merged**